### DISCUSSION TITLE
Where is link to most updated colab?

### DISCUSSION THREAD

**Discussion by grantbmiddleton** [TheLastBen/fast-stable-diffusion#628] [General]
This may be a dumb question, but where is the link to the most updated colab? I saw you said it was on the main page. Sorry I am new to Github.

**Comment by nawnie** [comment]
https://github.com/TheLastBen/fast-stable-diffusion/blob/main/fast-DreamBooth.ipynb

**Reply by nawnie** [reply]
or a direct colab link https://colab.research.google.com/github/TheLastBen/fast-stable-diffusion/blob/main/fast-DreamBooth.ipynb

**Reply by grantbmiddleton** [reply]
ok thank you. I wasn't sure how to open it in a google colab without a link.

**Reply by TheLastBen** [reply]
you can click on the Dreambooth thumbnail on the main page of the repo

**Reply by grantbmiddleton** [reply]
Oh wow... thank you
